.github/workflows/hexpm-mirrors.yml

```diff
@@ -35,24 +35,3 @@ jobs:
             https://mirror.invalid
             https://cdn.jsdelivr.net/hex
             https://builds.hex.pm
-
-  test-invalid-mirror:
-    name: Test errors with mirror
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use erlef/setup-beam
-        id: failing-setup-beam
-        uses: ./
-        continue-on-error: true
-        with:
-          otp-version: '24.3.4.6'
-          elixir-version: 'v1.13.4'
-          install-rebar: true
-          install-hex: true
-          hexpm-mirrors: https://mirror.invalid
-      - name: Report unexpected success
-        if: ${{ steps.failing-setup-beam.outcome == 'success' }}
-        run: |
-          echo "Action is expected to fail"
-          exit 1
```

Comment:
Why remove this?

Reply:
We get an annotation with an error, which lead us to think that there was an issue, but then there wasn't because the action is ✅. It's basically very misleading. We can add a non-workflow action test (e.g. JavaScript) to test the same condition, though, if it's absolutely necessary.